vpn/entrypoint.sh

```diff
@@ -30,6 +30,10 @@ if [ ! -d /etc/openvpn/proxy ]; then
     mkdir -p /etc/openvpn/proxy
 fi
 
+if [ ! -d /etc/openvpn/status ]; then
```

Comment:
If the directory is not cleaned upon restart, the `.vpn` files will mark a unit connected even if is not.
We could cleanup the directory on each start, or move it to a temporary location.